### ISSUE TITLE
Drag 'n Drop Problem

### ISSUE THREAD

**Issue by antekgla** [RandomEngy/VidCoder#805]
OS: Windows 10 Home 2004.

Hi, I have a problem dragging video files in Vidcoder.

I use the excellent file indexer search engine **Locate32** (32 bits old application not updated anymore) and I can't drag files in Vidcoder. (the forbidden icon appears when I drag a video file in Vidcoder)

First I tough a 32 vs 64 bits issue but after checking I discarded. I DONT HAVE PROBLEMS draging files from Locate32 to other windows 64 bits programs.  I can drag from Locate 32 search result windows to PotPlayer (64 bits) or Avidemux (64 bits).
Also I can drag files to Vidcoder from another 32 bits programs like FreeCommander XE

Later I think because I use a portable Vidcoder what have a 32 bits wrapper and a 64 bit executable (or something like that because in Task Manager appears **Vidcoder-6.30-Beta-Portable.exe** (32 bits) AND **Vidcoder.exe** (64 bits))
So I install Vidcoder with installer but the problem persist.

I more inclinated to think its a Locate32 fault because its a old program but like I say I don't have problems dragging files to other new programs so I think what some could be done in Vidcoder.

Any hints?
Thanks in advance.

**Comment by RandomEngy** [comment]
That's odd. I just use standard drag/drop functionality offered by WPF. Must be that framework code interacting with Locate32's code that's causing the problem. I don't know what the issue is, though I can offer another clue that running programs with different privilege levels can make the drag-drop operation fail.

**Comment by antekgla** [comment]
Not... both program are executed by the same user (me) so the privileges are the same for both programs.

**Comment by RandomEngy** [comment]
Yeah I think it's just some lower-level system code that is disallowing that drop operation. Sorry, don't have any special insight here.
